### PR TITLE
feat: added core changes to send drawable to actual device

### PR DESCRIPTION
**Pull request by adityastic**
Changes: 
- Created Separate Data Classes to Handle Bitmap Operations
- convertBitmap function is completely new and handles the data according to the bitmaps
- getFlash, getOptions, getSize, getMarquee are created according to the BitmapData now
- getMessages now calculates the ByteArray with a completely new logic. 
- All Functions are made to support bitmaps.

Screenshots:

NOTE: As I do not own the hardware yet, @yashk2000 can you please send the actual hardware screenshots to ensure that everything works. Please add it to the review itself

### CODE REVIEW

**Comment by adityastic** [comment]
@iamareebjamal @yashk2000 @yatri1609 @anhanh11001 Please Review

**Comment by adityastic** [comment]
Amazing @iamareebjamal. Have a look sir

**Comment by iamareebjamal** [comment]
> Amazing @iamareebjamal. Have a look sir

Are you praising me or yourself?

**Comment by adityastic** [comment]
> > Amazing @iamareebjamal. Have a look sir
> 
> Are you praising me or yourself?

I thought there might be some things that might not work. But it did, so I was happy

**Comment by adityastic** [comment]
you are an amazing mentor @iamareebjamal :)

**Comment by iamareebjamal** [comment]
I was joking

File: app/src/main/java/com/nilhcem/blenamebadge/device/model/BitmapDataToSend.kt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+package com.nilhcem.blenamebadge.device.model
+
+data class BitmapDataToSend(val messages: List<BitmapMessage>)
```
Comment (iamareebjamal) [inline app/src/main/java/com/nilhcem/blenamebadge/device/model/BitmapDataToSend.kt]: Why's this needed? Why not just use List<BitmapMessage> wherever required?

Comment (adityastic) [reply app/src/main/java/com/nilhcem/blenamebadge/device/model/BitmapDataToSend.kt]: I tried to follow the same pattern which was originally being used in sending text to the device

Comment (adityastic) [reply app/src/main/java/com/nilhcem/blenamebadge/device/model/BitmapDataToSend.kt]: @iamareebjamal I've Updated the PR with the new Layout changes, have a look

Comment (adityastic) [reply app/src/main/java/com/nilhcem/blenamebadge/device/model/BitmapDataToSend.kt]: @iamareebjamal btw sir, you can find the similar class for text here https://github.com/fossasia/badge-magic-android/blob/development/app/src/main/java/com/nilhcem/blenamebadge/device/model/DataToSend.kt

**Pull request merged**